src/server/app/admin/controllers/users/users.password.server.controller.js

```diff
@@ -219,9 +196,13 @@ exports.reset = (req, res, next) => {
 				html: emailHTML
 			};
 
-			smtpTransport.sendMail(mailOptions, (error) => {
-				done(error, 'done');
-			});
+			emailService.sendMail(mailOptions)
+				.then((result) => {
+					done(null);
+
+				}, (error) => {
+					logger.error({err: error, req: req}, 'Failure sending email.');
```

Comment:
Oops, I had a fix for this, but forgot to push it last night.  Let me know if its ok now.